solutions/observability/apps/monitor-fleet-managed-apm-server.md

```diff
@@ -99,8 +97,6 @@ See the [{{agent}} command reference](/reference/fleet/agent-command-reference.m
       period: 10s
       hosts: ["http://localhost:6791"]
       basepath: "/processes/apm-server-default"
```

Comment:
@lucabelluccini / @raultorrecilla can you confirm that the basepath needs to match the `component` id of the APM integration per https://github.com/elastic/observability-docs/pull/4749#discussion_r2145279469?